app/assets/stylesheets/pages/articles/article-detail/_detail.scss

```diff
@@ -0,0 +1,107 @@
+.detail{
+	margin-top: 37px;
+
+	&__wrapper{
+		width: 70%;
+
+		@media only screen and (max-width: 920px){
+			width: 80%;
+		}
+
+		@media only screen and (max-width: 640px){
+			width: 90%;
+		}
+
+		@media only screen and (max-width: 576px){
+			width: 100%;
+		}
+	}
+
+	&__title{
+		font-size: 32px;
+		line-height: 1.5;
+
+		@media only screen and (max-width: 1170px){
+			font-size: 28px;
+		}
+
+		@media only screen and (max-width: 767px){
+			font-size: 24px;
+		}
+
+		@media only screen and (max-width: 503px){
+			font-size: 20px;
+		}
+
+		@media only screen and (max-width: 440px){
+			font-size: 16px;
+		}
+
+		@media only screen and (max-width: 375px){
+			font-size: 15px;
+		}
+	}
+
+	&__author{
+		line-height: .8;
```

Comment:
try to not change the line-height but change the margin

Reply:
okok